lib/testutils/logrus_hook.go

```diff
@@ -59,3 +60,18 @@ func LogContains(logEntries []logrus.Entry, expLevel logrus.Level, expContents s
 	}
 	return false
 }
+
+// NewMemLogger creates a Logrus logger mocked by SimpleLogrusHook.
+func NewMemLogger() (*logrus.Logger, *SimpleLogrusHook) {
```

Comment:
This is not used anymore. Do we still want to keep it?

Reply:
The latest version uses it for checking the printed logline